third_party/txt/src/txt/font_collection.cc

```diff
@@ -202,6 +202,12 @@ std::shared_ptr<minikin::FontFamily> FontCollection::CreateMinikinFontFamily(
 const std::shared_ptr<minikin::FontFamily>& FontCollection::MatchFallbackFont(
     uint32_t ch,
     std::string locale) {
+  // Check if the ch's matched font has been cached. We cache the results of
```

Comment:
Rename the current `MatchFallbackFont` to be a helper function.  Then `MatchFallbackFont` can be changed to a wrapper that does a cache lookup and then calls the helper and inserts the cache entry if needed.  This will separate the cache handling from the font lookup.